tutorials/docker-gcplogs-driver/index.md

```diff
@@ -151,6 +151,70 @@ to use the Google Cloud logging driver when it is started by systemd:
     the [Logs Viewer](https://console.cloud.google.com/logs/viewer) as you did
     before.
 
+## Persisting configuration across reboots
+
+On Container-Optimized OS, files in `/etc/` are writable, but [data does not
```

Comment:
Please change to either:
"On a Container-Optimized OS"
or
"On Container-Optimized OSs"
Using the singular is preferred, since OSs is potentially confusing.